source/security/private-endpoints.txt

```diff
@@ -214,21 +188,161 @@ connection using either the App Services UI or the Admin API.
                  -H "Content-Type: application/json" \
                  -d '{
                    "cloud_provider_region": "<your-cloud-deployment-region>",
-                   "cloud_provider_endpoint_id": "<your-vpc-endpoint-id>"
+                   "cloud_provider_endpoint_id": "<your-vpc-endpoint-id>",
+                   "endpoint_type": "general",
+                   "comment": "<comment-about-endpoint>"
                  }'
 
-         .. step:: Modify Your VPC Private DNS
 
-            Use the AWS CLI to modify your VPC private DNS settings. This
-            ensures that your App can resolve the private endpoint's DNS name
-            to the correct IP address.
+Add a Sync Endpoint
+~~~~~~~~~~~~~~~~~~~
+
+Sync Endpoints require that you also have a General Endpoint associated with
+the Sync Endpoint.
 
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      .. procedure::
+
+         .. step:: Create a VPC Interface Endpoint
+
+            a. In the App Services UI, navigate to :guilabel:`App Settings` and
+               then click the :guilabel:`Private Endpoints` tab.
+            #. Click :guilabel:`Add Private Endpoint`, then click
+               :guilabel:`Sync`.
+            #. Add your VPC ID and Subnet IDs (comma-separated list) in their
+               separate fields, then click :guilabel:`Next`.
+            #. Copy the generated :guilabel:`VPC Interface Endpoint Command`.
+               The command should look something like this:
+            
+               .. code-block:: sh
+
+                 aws ec2 create-vpc-endpoint \
+                   --vpc-endpoint-type Interface
+                   --service-name <App Services Endpoint Service Name> \
+                   --vpc-id <your-vpc-id> \
+                   --region <your-aws-deployment-region> \
+                   --subnet-ids <your-subnet-ids>
+            #. In your terminal, create a Sync VPC Endpoint with the generated
```

Comment:
Ok, definitely confused about this. Are we creating AWS "VPC Endpoints" here? If so, should we be calling this a "VPC Endpoint" instead of a "Sync VPC Endpoint"? Are we using "Sync" and "General" here to make it clearer the relationship between the VPC Endpoint and whether App Services is using it as a "General" or "Sync" endpoint?

If the latter, I think we should be crystal clear that these are AWS VPC Endpoints that you will then add to App Services as either a "General" or "Sync" endpoint. And if I am totally misunderstanding this, maybe we can otherwise clarify the language? (Or feel free to disregard if this should be obvious to people who are more familiar with Private Endpoints.)

Reply:
Yeah, the language here isn't clear. It should be just VPC Endpoint. I think that's clear enough.